### PR TITLE
refactor for weekly cron job

### PR DESCRIPTION
**Pull request by harsham05**
@chrismattmann This is a working commit tested on **imagecat.d.o/solr/onionCore**

Thanks!

### CODE REVIEW

**Comment by harsham05** [comment]
@chrismattmann Also added File Dump functionality after scp'ing from Wrangler to @karanjeets script
Thank you @MJJoyce for batch_dump.sh !

**Comment by chrismattmann** [comment]
i think you forgot to add batch_dump.sh @harsham05

File: nutch/scripts/transfer-segments.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
```
Comment (chrismattmann) [inline nutch/scripts/transfer-segments.sh]: ALv2 header

File: nutch/scripts/wrangler_batch_dump.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```
Comment (chrismattmann) [inline nutch/scripts/wrangler_batch_dump.sh]: ALv2 header

**Comment by harsham05** [comment]
@chrismattmann I refactored batch_dump.sh as **wrangler_batch_dump.sh**

Added ALv2 headers.

**Comment by harsham05** [comment]
@chrismattmann Ready for Merge !

**Comment by chrismattmann** [comment]
where is dump_segments.sh @harsham05 ?

**Comment by harsham05** [comment]
@chrismattmann Added dump_segments.sh & all dependencies courtesy of @MJJoyce 
Thank you

**Comment by chrismattmann** [comment]
LGTM @harsham05

**Pull request merged**